go-controller/pkg/config/config.go

```diff
@@ -153,6 +153,7 @@ var (
 			V6OVNServiceHairpinMasqueradeIP: net.ParseIP("fd69::5"),
 		},
 		UserDefinedNetworkConntrackMarkBase: 2,
+		UserDefinedNetworkVRFTableBase:      2,
```

Comment:
do we need different config bases for mark v/s vrf ? if so why?

Reply:
table IDs are shared with the host and we need to document which range are we going to reserver for ovn-k, so we may enconter that some tableIDs are already reserved before this feature lands.

The ct-mark case has not such issue, that's why we differenciate.

Reply:
ah ack, @pperiyasamy ptal the VRFID generator bits

Reply:
EIP on secondary interfaces use custom routing [table](https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/controllers/egressip/egressip.go#L1323-L1325), this would be reused for secondary EIPs on user defined networks as well. so we may have to ensure this doesn't conflict with those tables.

Reply:
@pperiyasamy this means that https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/controllers/egressip/egressip.go#L1323-L1325 has to be integrated with this allocator to to have different one per network I think.

Reply:
yes @qinqon , we would need change it, so that EIP secondary NICs may use `MaxUserDefinedNetworks` as the base.

Reply:
@pperiyasamy I think it goes beyond that, so looks like we need per network a pair of table IDs one for VRF and another for egress IP, also looks like this last one we need one per interface also ?

Reply:
@tssurya thinking about it since we are going to create a vrf device per network maybe we can just use the link.Attrs().Index() from the vrf device to calculate routing table, that is going to be different per network.

Reply:
@qinqon yes, there would be one custom routing table per secondary interface, this can be derived with its ifIndex + base, we'll make sure it doesn't conflict with any other existing range.

another different question. Did you also think about what vrf name can be used for creating vrf device ? can we just use net name ?

Reply:
@tssurya suggested a really good idea which is the the management port + "vrf" at the end the management port is going to have the network name in it.

Reply:
ok, nice, that would work.